Tests/SPMTest/SPMTest.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,124 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "MapboxCommon",
+        "repositoryURL": "https://github.com/mapbox/mapbox-common-ios.git",
+        "state": {
+          "branch": null,
+          "revision": "50a87705438ee5b2c890a1fcacb54dfd867d738f",
+          "version": "10.0.2"
```

Comment:
update-version.sh runs `pod update` to ensure that the CocoaPods integration test’s Podfile.lock reflects the dependency versions that CocoaPods would resolve in downstream projects. What would be the equivalent for SPM, to ensure that we aren’t just testing a hard-coded set of resolved versions?

Reply:
According to [this documentation page](https://developer.apple.com/documentation/swift_packages/building_swift_packages_or_apps_that_use_them_in_continuous_integration_workflows), `xcodebuild` will always update all the dependencies to the latest versions compatible with the `Package.swift` files referenced in the `.pbxproj` file. In our case, this is [one file](https://github.com/mapbox/mapbox-navigation-ios/blob/main/Package.swift).
To ensure that `.pbxproj` will always reference the latest version of this `Package.swift` file, I use `sed` command to substitute current commit hash.
![image](https://user-images.githubusercontent.com/1976216/114862356-18978c00-9df7-11eb-8b97-06246b5a10fa.png)